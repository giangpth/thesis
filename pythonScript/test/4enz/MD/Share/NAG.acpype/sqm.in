Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
   6    C1       38.8480       81.5600       -3.1540 
   6    C2       38.2790       82.9940       -3.0870 
   6    C3       38.2880       83.6980       -4.4670 
   6    C4       39.6960       83.6130       -5.0800 
   6    C5       39.8930       82.1090       -5.2940 
   6    C6       41.1100       81.7840       -6.1580 
   6    C7       36.5380       83.8740       -1.5570 
   6    C8       35.1200       83.7790       -1.0830 
   7    N2       36.9260       83.0110       -2.5240 
   8    O3       37.8980       85.0350       -4.2970 
   8    O4       39.9050       84.4280       -6.2360 
   8    O5       40.0040       81.4960       -4.0090 
   8    O6       42.2990       82.1460       -5.4880 
   8    O7       37.2660       84.7200       -1.0470 
   1   H11       38.0810       80.8910       -3.5440 
   1   H12       39.1270       81.2380       -2.1510 
   1    H2       38.9200       83.5710       -2.4200 
   1    H3       37.5790       83.1990       -5.1280 
   1    H4       40.4070       83.9330       -4.3180 
   1    H5       39.0060       81.7130       -5.7880 
   1   H61       41.1270       80.7140       -6.3670 
   1   H62       41.0430       82.3340       -7.0970 
   1   HN2       36.2510       82.3470       -2.8750 
   1   HO3       37.9000       85.4800       -5.1480 
   1   HO4       40.8450       84.5850       -6.3510 
   1   HO6       43.0550       81.9370       -6.0410 
   1   H81       34.9440       84.5260       -0.3090 
   1   H82       34.9390       82.7850       -0.6750 
   1   H83       34.4450       83.9570       -1.9200 

