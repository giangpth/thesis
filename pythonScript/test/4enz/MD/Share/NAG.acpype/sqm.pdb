ATOM      1  C1  NAG  1101      38.691  81.770  -3.296  1.00  0.00           C
ATOM      2  C2  NAG  1101      38.154  83.219  -3.354  1.00  0.00           C
ATOM      3  C3  NAG  1101      38.098  83.644  -4.842  1.00  0.00           C
ATOM      4  C4  NAG  1101      39.503  83.484  -5.447  1.00  0.00           C
ATOM      5  C5  NAG  1101      39.983  82.036  -5.276  1.00  0.00           C
ATOM      6  C6  NAG  1101      41.427  81.852  -5.750  1.00  0.00           C
ATOM      7  C7  NAG  1101      36.759  83.283  -1.324  1.00  0.00           C
ATOM      8  C8  NAG  1101      35.439  83.691  -0.717  1.00  0.00           C
ATOM      9  N2  NAG  1101      36.869  83.356  -2.705  1.00  0.00           N
ATOM     10  O3  NAG  1101      37.677  84.996  -4.894  1.00  0.00           O
ATOM     11  O4  NAG  1101      39.380  83.823  -6.819  1.00  0.00           O
ATOM     12  O5  NAG  1101      39.980  81.690  -3.890  1.00  0.00           O
ATOM     13  O6  NAG  1101      42.292  82.820  -5.197  1.00  0.00           O
ATOM     14  O7  NAG  1101      37.695  82.850  -0.623  1.00  0.00           O
ATOM     15  H11 NAG  1101      37.989  81.061  -3.805  1.00  0.00           H
ATOM     16  H12 NAG  1101      38.863  81.462  -2.228  1.00  0.00           H
ATOM     17  H2  NAG  1101      38.909  83.881  -2.828  1.00  0.00           H
ATOM     18  H3  NAG  1101      37.370  83.005  -5.415  1.00  0.00           H
ATOM     19  H4  NAG  1101      40.225  84.183  -4.937  1.00  0.00           H
ATOM     20  H5  NAG  1101      39.307  81.340  -5.845  1.00  0.00           H
ATOM     21  H61 NAG  1101      41.777  80.819  -5.488  1.00  0.00           H
ATOM     22  H62 NAG  1101      41.483  82.009  -6.860  1.00  0.00           H
ATOM     23  HN2 NAG  1101      36.197  83.897  -3.200  1.00  0.00           H
ATOM     24  HO3 NAG  1101      37.805  85.289  -5.807  1.00  0.00           H
ATOM     25  HO4 NAG  1101      40.278  83.910  -7.162  1.00  0.00           H
ATOM     26  HO6 NAG  1101      42.261  82.712  -4.237  1.00  0.00           H
ATOM     27  H81 NAG  1101      35.488  83.531   0.388  1.00  0.00           H
ATOM     28  H82 NAG  1101      34.607  83.075  -1.136  1.00  0.00           H
ATOM     29  H83 NAG  1101      35.234  84.769  -0.919  1.00  0.00           H
