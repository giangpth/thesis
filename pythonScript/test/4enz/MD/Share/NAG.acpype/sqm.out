            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 41

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =       71
| QMMM:    diag iterations used for timing =       15
| QMMM:
| QMMM:              Internal diag routine = 0.091074 seconds
| QMMM:                 Dspev diag routine = 0.199898 seconds
| QMMM:                Dspevd diag routine = 0.114245 seconds
| QMMM:                Dspevx diag routine = 0.391843 seconds
| QMMM:                 Dsyev diag routine = 0.071325 seconds
| QMMM:                Dsyevd diag routine = 0.154179 seconds
| QMMM:                Dsyevr diag routine = 0.125420 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.014923 seconds
| QMMM:
| QMMM: Using dsyev routine (diag_routine=5).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       38.8480   81.5600   -3.1540
  QMMM:     2        2      C       38.2790   82.9940   -3.0870
  QMMM:     3        3      C       38.2880   83.6980   -4.4670
  QMMM:     4        4      C       39.6960   83.6130   -5.0800
  QMMM:     5        5      C       39.8930   82.1090   -5.2940
  QMMM:     6        6      C       41.1100   81.7840   -6.1580
  QMMM:     7        7      C       36.5380   83.8740   -1.5570
  QMMM:     8        8      C       35.1200   83.7790   -1.0830
  QMMM:     9        9      N       36.9260   83.0110   -2.5240
  QMMM:    10       10      O       37.8980   85.0350   -4.2970
  QMMM:    11       11      O       39.9050   84.4280   -6.2360
  QMMM:    12       12      O       40.0040   81.4960   -4.0090
  QMMM:    13       13      O       42.2990   82.1460   -5.4880
  QMMM:    14       14      O       37.2660   84.7200   -1.0470
  QMMM:    15       15      H       38.0810   80.8910   -3.5440
  QMMM:    16       16      H       39.1270   81.2380   -2.1510
  QMMM:    17       17      H       38.9200   83.5710   -2.4200
  QMMM:    18       18      H       37.5790   83.1990   -5.1280
  QMMM:    19       19      H       40.4070   83.9330   -4.3180
  QMMM:    20       20      H       39.0060   81.7130   -5.7880
  QMMM:    21       21      H       41.1270   80.7140   -6.3670
  QMMM:    22       22      H       41.0430   82.3340   -7.0970
  QMMM:    23       23      H       36.2510   82.3470   -2.8750
  QMMM:    24       24      H       37.9000   85.4800   -5.1480
  QMMM:    25       25      H       40.8450   84.5850   -6.3510
  QMMM:    26       26      H       43.0550   81.9370   -6.0410
  QMMM:    27       27      H       34.9440   84.5260   -0.3090
  QMMM:    28       28      H       34.9390   82.7850   -0.6750
  QMMM:    29       29      H       34.4450   83.9570   -1.9200

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -235.8181 kcal/mol        1.3229 kcal/(mol*A)
xmin    20     -238.6741 kcal/mol        0.4899 kcal/(mol*A)
xmin    30     -239.4850 kcal/mol        0.6342 kcal/(mol*A)
xmin    40     -239.7817 kcal/mol        0.6419 kcal/(mol*A)
xmin    50     -239.9539 kcal/mol        0.2953 kcal/(mol*A)
xmin    60     -240.0082 kcal/mol        0.0401 kcal/(mol*A)
xmin    70     -240.0108 kcal/mol        0.0084 kcal/(mol*A)
xmin    80     -240.0108 kcal/mol        0.0016 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -41.6106  -39.5903  -38.6776  -38.0107  -37.2377  -35.3295  -32.0434  -31.5806
 -29.8110  -27.2881  -25.0070  -23.3104  -21.6382  -21.3049  -19.4537  -18.6790
 -18.2367  -17.7050  -17.6079  -17.3735  -16.8126  -16.1421  -15.8668  -15.6301
 -15.1192  -15.0305  -14.8593  -14.4182  -13.9684  -13.7730  -13.5851  -13.0306
 -12.8184  -12.5460  -12.3290  -11.8168  -11.3607  -11.2305  -10.8875  -10.7878
 -10.0511    1.2656    1.8837    2.5697    2.7966    2.9746    3.0135    3.2158
   3.3728    3.4855    3.5088    3.7015    3.7419    3.7709    3.9976    4.0203
   4.0815    4.1016    4.1150    4.3171    4.4420    4.4889    4.5272    4.6144
   5.1867    5.6516    5.8912    5.9693    6.0945    6.1565    6.9674

 Heat of formation   =       -240.01083256 kcal/mol  (      -10.40765069 eV)

 Total SCF energy    =     -70146.29519666 kcal/mol  (    -3041.77161427 eV)
 Electronic energy   =    -385684.14704543 kcal/mol  (   -16724.51962384 eV)
 Core-core repulsion =     315537.85184876 kcal/mol  (    13682.74800957 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      C                 -0.034
     2      C                 -0.001
     3      C                  0.022
     4      C                 -0.029
     5      C                 -0.046
     6      C                 -0.018
     7      C                  0.295
     8      C                 -0.242
     9      N                 -0.364
    10      O                 -0.334
    11      O                 -0.334
    12      O                 -0.271
    13      O                 -0.332
    14      O                 -0.368
    15      H                  0.079
    16      H                  0.148
    17      H                  0.114
    18      H                  0.095
    19      H                  0.114
    20      H                  0.100
    21      H                  0.089
    22      H                  0.106
    23      H                  0.230
    24      H                  0.225
    25      H                  0.218
    26      H                  0.218
    27      H                  0.117
    28      H                  0.102
    29      H                  0.100
 Total Mulliken Charge =       0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE    -2.188    0.009   -2.219    3.116
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      C       38.6914   81.7705   -3.2963
  QMMM:     2        2      C       38.1544   83.2191   -3.3540
  QMMM:     3        3      C       38.0984   83.6442   -4.8420
  QMMM:     4        4      C       39.5032   83.4842   -5.4466
  QMMM:     5        5      C       39.9827   82.0357   -5.2762
  QMMM:     6        6      C       41.4270   81.8522   -5.7497
  QMMM:     7        7      C       36.7591   83.2834   -1.3236
  QMMM:     8        8      C       35.4392   83.6906   -0.7165
  QMMM:     9        9      N       36.8694   83.3564   -2.7049
  QMMM:    10       10      O       37.6773   84.9960   -4.8940
  QMMM:    11       11      O       39.3795   83.8233   -6.8187
  QMMM:    12       12      O       39.9799   81.6896   -3.8905
  QMMM:    13       13      O       42.2922   82.8203   -5.1968
  QMMM:    14       14      O       37.6951   82.8499   -0.6230
  QMMM:    15       15      H       37.9889   81.0606   -3.8050
  QMMM:    16       16      H       38.8627   81.4619   -2.2283
  QMMM:    17       17      H       38.9086   83.8812   -2.8281
  QMMM:    18       18      H       37.3696   83.0049   -5.4151
  QMMM:    19       19      H       40.2247   84.1826   -4.9366
  QMMM:    20       20      H       39.3066   81.3399   -5.8451
  QMMM:    21       21      H       41.7767   80.8185   -5.4882
  QMMM:    22       22      H       41.4825   82.0087   -6.8600
  QMMM:    23       23      H       36.1973   83.8971   -3.2004
  QMMM:    24       24      H       37.8050   85.2888   -5.8070
  QMMM:    25       25      H       40.2779   83.9099   -7.1623
  QMMM:    26       26      H       42.2606   82.7117   -4.2368
  QMMM:    27       27      H       35.4881   83.5308    0.3882
  QMMM:    28       28      H       34.6073   83.0753   -1.1362
  QMMM:    29       29      H       35.2340   84.7692   -0.9193

           --------- Calculation Completed ----------

